Lexplorer/Pages/Account.razor

```diff
@@ -121,5 +116,11 @@
             <TransactionTableDetails TransactionData=@context ignoreUserIDForLink=@accountId />
             <MudTd DataLabel="Timestamp">@context.verifiedAt</MudTd>
         </RowTemplate>
+        <PagerContent>
+            @if (transactions?.Count() > 10)
+            {
+                <MudTablePager HideRowsPerPage="true" HidePageNumber="false" InfoFormat="@("{first_item}-{last_item} of {all_items}")" HorizontalAlignment="HorizontalAlignment.Left" />
+            }
+        </PagerContent>
```

Comment:
How's this going to work? We're *not* getting all transactions - we're getting them in pages of 25, manually. Getting them all is simply not possible, imagine a pool with thousands and thousands of transactions.

Reply:
Was not aware of this, that's why reviews are great :) Can change it back

Reply:
Changed it back :)